packages/Uploader/src/Uploader.js

```diff
@@ -96,6 +100,7 @@ const defaultProps = {
   onProcessed: () => {},
   onRequest: null,
   supportedMimeTypes: ["*/*"],
+  zIndex: 1,
```

Comment:
This could probably be `null` since the `<Popover>` already defines a default value for its `zIndex`:
https://github.com/acl-services/paprika/blob/master/packages/Popover/src/Popover.js#L572

But the effect is the same, so 🤷‍♂️

Reply:
😂 🤷‍♂️